tests/e2e/local/minikube/setup_host.sh

```diff
@@ -14,12 +14,14 @@ case "${OSTYPE}" in
   *) echo "unsupported: ${OSTYPE}" ;;
 esac
 
-# shellcheck disable=SC2153
-if [ ! -z "$VM_DRIVER" ]; then
-  vm_driver="$VM_DRIVER"
-fi
+# Virtual machine driver, the default value is decided by your OS type if it's not specified,
+# e.g.:
+#   hyperkit default to darwin os
+#   kvm2 default to Debian or Ubuntu os
+# Besides, you can set any vm-driver you like via exporting `VM_DRIVER`for you environment.
```

Comment:
typo: your -> your

Reply:
nit: should put in a white space between words `VM_DRIVER` and for.

Reply:
fixed